extension/src/shared/ui-components/FeedBackModal.tsx

```diff
@@ -0,0 +1,156 @@
+import { FormControl } from "@mui/base/FormControl";
+import { Input, inputClasses } from "@mui/base/Input";
+import { TextareaAutosize } from "@mui/base/TextareaAutosize";
+import CancelIcon from "@mui/icons-material/Cancel";
+import {
+  Box,
+  Button,
+  Checkbox,
+  FormControlLabel,
+  IconButton,
+  Link,
+  Modal,
+  Typography,
+  styled,
+} from "@mui/material";
+import { useAtom } from "jotai";
+import { useState } from "react";
+
+import { HelperText } from "../../prototypes/ui-components/HelperText";
+import { Label } from "../../prototypes/ui-components/InputLabel";
+import { showFeedbackModalAtom } from "../../prototypes/view/states/app";
+
+const style = {
+  position: "absolute" as const,
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  width: 420,
+  bgcolor: "background.paper",
+  color: "#000",
+  margin: "auto",
+  padding: "18px 22px",
+};
+
+const FeedBackModal: React.FC = () => {
+  const [showFeedbackModal, setShowFeedbackModal] = useAtom(showFeedbackModalAtom);
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [comment, setComment] = useState("");
+  const [attachMapReview, setAttachMapReview] = useState(false);
+
+  const handleSubmit = () => {
+    setName("");
+    setEmail("");
+    setComment("");
+    setAttachMapReview(false);
+    setShowFeedbackModal(false);
+  };
```

Comment:
I would like to keep ui-components be reusable UI as I mentioned above comment. So could you do update this as well like below?

```tsx
const FeedBack = () => {
  const handleSubmit = (params: { name, email, comment, ... }) => {
  };
  return <FeedBackModal onSubmit={handleSubmit} />
}
```

Reply:
Also this function should be like this.
```suggestion
  const handleSubmit = async () => {
    try {
        await onSubmit();
        setName("");
        setEmail("");
        setComment("");
        setAttachMapReview(false);
        setShowFeedbackModal(false);
    } catch {
      // Error handling
    }
  };
```